go.mod

```diff
@@ -1,23 +1,23 @@
 module github.com/docker/compose/v2
 
-go 1.21.0
+go 1.22.0
```

Comment:
Did it automatically update this one, or did it still allow go 1.21.0 here?

I generally try to treat this one the same as other dependencies; list the minimum required version, and only update if it's _impossible_ to use with older versions; see https://github.com/containerd/containerd/pull/10596#discussion_r1721294997